AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -322,6 +323,14 @@ public static boolean hasReceivedDiffs() {
     return hasReceivedDiffs;
   }
 
+  public static Map<String, Object> getVariantDebugInfo() {
```

Comment:
Create and use a class to represent this info such as `VariantDebugInfo` which could contain a collection of `AbTestVariableOverride` or something similar.

Reply:
I do not think we need to create a class for this. This is a special one-off feature and only to be used for debugging. Creating its own class and taking on the serializing from JSON response as well as serializing / deserializing for persistence is extra work that is not needed for this feature

Reply:
It's a good idea to create new classes for data structures such as this as it makes SDK much more user-friendly to use and brings up the quality of code.  Instead of having yourself or customer use an opaque Map class, provide them with self documenting data structure, even if its for debugging.

It should not be much extra work, but would increase code quality by a bit.

It should be done across the board for other things such as `regions`, but can be done in separate review.